lib/Conversion/TorchToTMTensor/TorchToTMTensor.cpp

```diff
@@ -523,6 +548,50 @@ class ConvertAtenMaxPool2dWithIndicesBackwardOp
 };
 } // namespace
 
+namespace {
+class ConvertAtenCumsumOp : public OpConversionPattern<AtenCumsumOp> {
```

Comment:
This op also takes an optional `dtype` argument, which is currently not being handled here

https://pytorch.org/docs/stable/generated/torch.cumsum.html